django/core/files/move.py

```diff
@@ -66,7 +67,15 @@ def file_move_safe(old_file_name, new_file_name, chunk_size=1024 * 64, allow_ove
         finally:
             locks.unlock(fd)
             os.close(fd)
-    copystat(old_file_name, new_file_name)
+
+    try:
+        copystat(old_file_name, new_file_name)
+    except PermissionError as e:
+        # Certain filesystems (CIFS) fail to copy the files metadata if
+        # the destination filesystem is not the same type of filesystem
+        # as the source, ignore it.
+        if getattr(e, 'errno', 0) != errno.EPERM:
```

Comment:
@charettes It is likely overly cautious in the way we were treating it, but we were not sure about `errno` so we mimic'ed the way `winerr` was handled in the method just a few line below this.

Based on the documentation, it appears that the `errno` attribute is should be present for any exception that is subclassed from `OSError`. I would welcome any feedback from people with a more definitive answer.